ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -143,6 +139,33 @@ const FreeCreditsUsage = ({isAboveLimit, usage}: FreeCreditsProps) => {
   </React.Fragment>;
 };
 
+interface ExpirationProps {
+  modules: Array<AccessModuleStatus>;
+  UserStatusComponent: () => JSX.Element;
+}
+const AccessModuleExpirations = ({modules, UserStatusComponent}: ExpirationProps) => {
+  const {enableComplianceTraining} = serverConfigStore.get().config;
```

Comment:
Why is compliance training special cased here? Can you add a comment?

Reply:
sure - it's the only one with a feature flag